Tests/TestLogs.cs

```diff
@@ -49,7 +50,7 @@ public void TestAccents()
             Assert.IsTrue(legality.IsCardLegal(CardName.FixAccents("DandAþn")));
             Assert.IsFalse(legality.IsCardLegal(CardName.FixAccents("Lim-dl")));
         }
-
+        
```

Comment:
You should install the https://editorconfig.org/ plugin for your preferred IDE

Reply:
Wait, nevermind, the `.editorconfig` file isn't actually in this repo.  I'll fix that for future PRs :)